busy_beaver/blueprints/tasks/__init__.py

```diff
@@ -0,0 +1,18 @@
+from flask import blueprints
+
+from .github_summary import PublishGitHubSummaryResource
+from .retweeter import TwitterPollingResource
+from busy_beaver.decorators import authentication_required
+
+tasks_bp = blueprints.Blueprint("tasks", __name__)
+admin_role_required = authentication_required(roles=["admin"])
```

Comment:
As we are using class-based views, this is how we have to decorate views to add token-based authentication. I am not a fan, but the workflow is completely tested and works exactly as expected.

Being pragmatic and getting things working in the way they were before. We can always add a flask plugin to manage our API User accounts and roles.